shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -1,10 +1,7 @@
 package com.netguru.codereview.network.model
 
-import kotlinx.serialization.SerialName
-import kotlinx.serialization.Serializable
-
 data class ShopListResponse(
-    val list_id: String,
-    val userId: Int,
-    val listName: String
+    val listId: String,
```

Comment:
name variables using camel style, and to make it consistent, except in case of returned from back-end response this is regarding entity modules